packages/core/src/lib/implementation/execute-plugin.ts

```diff
@@ -105,18 +112,33 @@ export async function executePlugin(
       ...(description && { description }),
       ...(docsUrl && { docsUrl }),
       ...(groups && { groups }),
-    };
-    return pluginReport;
+    } satisfies PluginReport;
   } catch (error) {
     const e = error as Error;
     throw new PluginOutputError(slug, e);
   }
 }
 
+const MOCK_PROGRESS: ProgressBar = {
+  incrementInSteps: (_: number) => {
+    _;
+    void 0;
+  },
+  updateTitle: (_: string) => {
+    _;
+    void 0;
+  },
+  endProgress: (_?: string) => {
+    _;
+    void 0;
+  },
+};
+
 /**
  * Execute multiple plugins and aggregates their output.
  * @public
- * @param plugins array of {@link PluginConfig} objects
+ * @param plugins array of {@param options
+@link PluginConfig} objects
```

Comment:
Probably, there is a typo.

```suggestion
 * @param plugins array of {@link PluginConfig} objects
 * @param {Object} [options] execution options
 * @param {boolean} options.progress show progress bar
```

Reply:
adressed